docs/NNNN-n-tuple-omit-prefix-storage-layout.md

```diff
@@ -0,0 +1,183 @@
+# OCFL Community Extension NNNN: N Tuple Omit Prefix Storage Layout
+
+  * **Extension Name:** NNNN-n-tuple-omit-prefix-storage-layout
+  * **Authors:** Michael Vandermillen and Andrew Woods
+  * **Minimum OCFL Version:** 1.0
+  * **OCFL Community Extensions Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+## Overview
+
+This storage root extension describes an OCFL storage layout combining a pairtree-like root directory structure derived from prefix-omitted object identifiers, followed by the prefix-omitted object identifier themselves. 
+The OCFL object identifiers are expected to contain prefixes which are removed in the mapping to directory names. 
+The OCFL object identifier prefix is defined as all characters before and including a configurable delimiter.
+Where the prefix-omitted identifier length is less than tuple size * number of tuples, the remaining object id (prefix omitted) is left or right-side, zero-padded (configurable, left default), and optionally reversed (default false).
+The object id is then divided into N n-tuple segments, and used to create nested paths under the OCFL storage root, followed by the prefix-omitted object id directory.  
+
+This layout combines the advantages of 0006-flat-omit-prefix-storage-layout (directory name transparency) and the 0004-hashed-n-tuple-storage-layout (enhanced file system/bucket performance). 
+
+The limitations of this layout are filesystem dependent (with one exception), and are generally as follows:
+
+* The size of object identifiers, minus the length of the prefix, cannot exceed the maximum allowed directory name size
+  (eg. 255 characters)
+* Object identifiers cannot include characters that are illegal in directory names
+* Only ASCII characters are supported.
```

Comment:
I suggest change from:
> Only ASCII characters are supported.

to
> This extension is defined over the ASCII subset of UTF-8 (code points 0x20 to 0x7F). And character outside of this range in either an identifier or a path is an error.

Reply:
OK, I made this change and pushed, thanks!